src/components/PWAInstructions/index.tsx

```diff
@@ -118,12 +119,21 @@ const devices = {
   },
 };
 
-const PWAInstructions = (props) => {
-  const [device, setDevice] = useState(null);
-  const [platform, setPlatform] = useState(null);
+type props = {
```

Comment:
Type names should be PascalCase (i.e. start with an uppercase letter).
```suggestion
type Props = {
```